samples/sinks.js

```diff
@@ -142,21 +145,23 @@ function updateSink (sinkName, filter) {
    * filter information.
    */
   const metadata = {
-    filter: filter
+    filter: filter,
   };
 
   // See https://googlecloudplatform.github.io/google-cloud-node/#/docs/logging/latest/logging/sink?method=setMetadata
-  sink.setMetadata(metadata)
-    .then((results) => {
-      console.log(`Sink ${sinkName} updated.`);
+  sink
+    .setMetadata(metadata)
+    .then(results => {
+      const metadata = results[0];
+      console.log(`Sink ${sinkName} updated:`, metadata);
```

Comment:
@jmdobry I modified this line so I wouldn't have to remove `results` from the callback arguments. I can revert if you think it was better before.

Reply:
What you did is fine.